Simplenote/metadata/release_notes.txt

```diff
@@ -1 +1,4 @@
-* Fixed a bug with duplicate tags and special characters in tag names
+* Added improvements for devices with hardware keyboards
+* Added support for screen readers when adding a new note
+* Fixed an issue with font size resizing when swiping
```

Comment:
```suggestion
```

Same comment about this item in the release notes if we decide to remove it from the `RELEASE_NOTES.txt`.